public/app/routes/GrafanaCtrl.ts

```diff
@@ -58,6 +58,13 @@ export class GrafanaCtrl {
     setKeybindingSrv(keybindingSrv);
     setDashboardSrv(dashboardSrv);
 
+    locationUtil.initialise({
+      getConfig: () => config,
+      getTimeRangeForUrl: getTimeSrv().timeRangeForUrl,
+      // @ts-ignore
+      buildParamsFromVariables: getTemplateSrv().fillVariableValuesForUrl,
+    });
```

Comment:
locationUtils depend on some runtime dependencies. Since I need those in data links supplier attached via field overrides, I need to provide the runtime deps somehow :/